src/test/scala/uk/gov/hmrc/ui/pages/BasePage.scala

```diff
@@ -16,6 +16,27 @@
 
 package uk.gov.hmrc.ui.pages
 
+import org.openqa.selenium.{By, WebDriver}
 import uk.gov.hmrc.selenium.component.PageObject
+import uk.gov.hmrc.selenium.webdriver.Driver
+import scala.jdk.CollectionConverters.CollectionHasAsScala
 
-trait BasePage extends PageObject {}
+trait BasePage extends PageObject {
+
+  protected val url: String
+
+  def webDriver: WebDriver = Driver.instance
+
+  def goTo(): Unit = get(url)
+
+  def pageTitle(): String = webDriver.getTitle
```

Comment:
By using the ui-test-runner PageObject.scala, there are a few helpers which could simplify some of the code. This is absolutely optional and down to teams to decide if they want to use it or not :-) 

For example, here you could consider:

`def pageTitle(): String = getTitle`

Reply:
Done